charts/backup-operator/templates/deployment.yaml

```diff
@@ -25,10 +25,10 @@ spec:
       containers:
       - args:
         - --enable-leader-election
+        - --worker-image {{ .Values.image.repository }}:{{ .Values.image.tag }}
```

Comment:
The space is not allowed here AFAIK, you have to replace it with an equal sign. The rest is fine.